### PR TITLE
feat:login and signup added

### PR DESCRIPTION
**Pull request by ArpitPatni**
Please review my login and signup page.

### CODE REVIEW

File: Styles/login.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    border: none;
+}
+
+.container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  background-color: var(--white-color);
+  transform: translate(-50%, -50%);
+  width: 410px;
```
Comment (harshitpaliwal95) [inline Styles/login.css:38]: use rem as units

**Review by harshitpaliwal95 — COMMENTED** [review]
- overall looks good 
- one suggestion add some padding to input

**Review by kirti2602 — APPROVED** [review]
Looks fine. Can merge

File: Styles/login.css (hunk 1 above)
Comment (ArpitPatni) [reply Styles/login.css:38]: Thank you for suggestion!

**Pull request merged**